### PR TITLE
gordonz - who

### LINKED ISSUE #75: who
I would like to do the who command

### PR DESCRIPTION
**Pull request by g-zhang**
# Advanced HW 9 submission

Closes #75

Name: Gordon Zhang
Uniqname: gordonz

### CODE REVIEW

**Comment by mterwill** [comment]
I think you have some whitespace issues that doesn't conform to Markdown spec.

screen shot 2016-03-21 at 5 27 40 pm

File: _commands/basics/who.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+           system boot  2016-03-18 13:04
+           run-level 5  2016-03-18 13:04
+LOGIN      tty1         2016-03-18 13:04              1594 id=tty1
+           pts/0        2016-03-19 01:59             12031 id=ts/0  term=0 exit=0
+alice    + pts/1        2016-03-19 22:38   .          2715 (:1)
+bob      + pts/2        2016-03-19 22:39  old         5239 (:1)
+mallory  - pts/3        2016-03-19 22:40   .         10892 (0587396660.track.nsa.gov)
+~~~
+##### Break it down
+`who` with the `-h` flag prints out a line of headings for each column of information. The `-a` or the `-all` option prints all the data the `who` command allows. This includes the boot time, login processes, current runlevel, dead processes, and other user & processes data.
```
Comment (mterwill) [inline _commands/basics/who.md]: `-h` or `-H`? be consistent.

**Comment by mterwill** [comment]
Would like to see some more explanation on stuff people might not have seen before. What's `pts/0`?

File: _commands/basics/who.md (hunk 1 above)
Comment (g-zhang) [inline _commands/basics/who.md]: Yeah that's a typo, I meant `-H`

**Comment by g-zhang** [comment]
I can't seem to see the whitespace issue you're talking about. It looks fine from the github website and all my computers. Could you please elaborate on that?

**Comment by mterwill** [comment]
Did you get the jekyll server up and running to test? Your page doesn't format correctly (see the picture I posted in the first comment).

**Comment by g-zhang** [comment]
Ah I see, I didn't get what that picture was of at first. I'll check it out

**Comment by g-zhang** [comment]
I just pushed a new version; I tried following a similar format of breakdown like the ls page. Let me know what you think, or if there any other issues

**Comment by mterwill** [comment]
Looks good!

**Pull request merged**